MaterialLib/Fluid/Viscosity/VogelsLiquidDynamicViscosity.h

```diff
@@ -82,7 +83,8 @@ class VogelsLiquidDynamicViscosity final : public FluidProperty
 };
 
 /**  Parameters A, B, C.
- *  <a href="ddbst"> http://ddbonline.ddbst.de/VogelCalculation/VogelCalculationCGI.exe</a>
+ *  <a href="ddbst">
+ * http://ddbonline.ddbst.de/VogelCalculation/VogelCalculationCGI.exe</a>
```

Comment:
The link should be generated via
<a href="http://ddbonline.ddbst.de/VogelCalculation/VogelCalculationCGI.exe">ddbst</a>